hs_communities/static/js/dual-listbox.js

```diff
@@ -0,0 +1,177 @@
+                isValid = true
+            }
+
+            if (!isValid) {
+                this.$data.errmsg = "Must make a valid selection for all components"
+            }
+
+            if (this.subtopic.includes("--") || this.location.includes("--")) {
+                this.$data.errmsg = "Entries may not contain a double hyphen";
+                isValid = false;
+            }
+            return isValid
+        }
+    }
+});
+
+function isValidYear(n) {
+    if (n.length === 0 || n === "Ongoing") {
+        return true
+    } else {
+        let validYear = (!isNaN(parseFloat(n)) && isFinite(n));
+        return validYear && n.length <= 4
+    }
+}
+
+// When a CZO User clicks on the Resource Title show the Title Assistant modal and prepopulate UI
+function titleClick() {
+    (function () {
+        /*
+        Optional text might be provided, affecting the Array index of what is stored
+
+        0: CZO selection
+        1: Topics
+        2: Optional text (or Location)
+        3: Location (or Dates if Optional text was provided in 2)
+        4: Dates (if Optional text was provided in 2)
+        */
+        let sections = $("#txt-title").val().split(" -- ");
+        let topics;
+        let yearsSection;
+
+        if (!this.$data.title) {  // don't repopulate modal if user never browsed away; UI will already contain previous selections
+            if (sections.length === 5) { // title contains optional subtopic
+                this.$data.regionSelected = sections[0];
+                topics = sections[1].split(",");
+                this.$data.subtopic = sections[2];
+                this.$data.location = sections[3];
+                yearsSection = sections[4]
+            } else if (sections.length === 4) {
+                this.$data.regionSelected = sections[0];
+                topics = sections[1].split(",");
+                this.$data.location = sections[2];
+                yearsSection = sections[3];
+            }
+
+            if (sections.length === 4 || sections.length === 5) {  // only 4 or 5 sections accepted; no other quantity is valid
+                topics.forEach(function (topic) {
+                    this.$data.topics.selectedItems.push({value: topic.trim(), displayValue: topic.trim(), isSelected: false});
+                }.bind(this));
+
+                yearsSection = yearsSection.substring(1, yearsSection.length - 1).split("-");
+                this.$data.startYear = yearsSection[0];
+                this.$data.endYear = yearsSection[1];
+                if (this.$data.endYear === "Ongoing") {
+                    $("#end-date-ongoing").prop("checked", true)
+                }
+                this.itemMoved();
+            }
+        }
+        if (!this.subtopic) {
+            this.$data.subtopic = 'clear';
+            this.$data.subtopic = '';
+        }
+        $("#title-modal").modal('show');
+    }.bind(TitleAssistantApp)());
```

Comment:
@Maurier note use of `$` in jQuery call

Reply:
I think this should be fine. I've been doing this myself when I need to reference content that has not been wrapped with a Vue component.